app/src/main/java/protect/card_locker/preferences/SettingsActivity.java

```diff
@@ -14,6 +14,7 @@
 import java.util.Locale;
 
 import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
```

Comment:
This import doesn't seem to be used.

Reply:
Fixed in e4f2f8f. Android Studio loves to silently add stuff to imports to help me and then I forgot them when I change the code because I didn't notice them getting added. Wouldn't surprise me if more classes suffer from this.